src/cli/types.ts

```diff
@@ -4,15 +4,4 @@ export type Results = {
   [key in keyof ResultMap]: number[];
 };
 
-export type ResultSet = {
-  // name of the containing folder
-  name: string;
-  // client specific results
-  client: Results;
-  // server specific results
-  server: Results;
-  // interactions specific results
-  interactions: Results;
-  // number of result files in the directory
-  results: number;
-};
+export type ResultByGroupId = { [groupId: string]: Results };
```

Comment:
can we tighten the type here for groupId. Does this type already exist for example in the src elsewhere?